Segment-Amplitude.podspec

```diff
@@ -21,6 +21,6 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
 
-  s.dependency 'Analytics', '~> 3.2' # support for iOS 7 +
+  s.dependency 'Analytics', '~> 3.6' # support for iOS 7 +
```

Comment:
This will require using thew newest version of this integration to upgrade analytics-ios too. Not a big deal, but might be surprising for some folks. Something to keep in mind for next time.